x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/actions/types.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SearchSessionStatus } from '../../../../../common';
+
+export type OnActionComplete = () => void;
+
+export enum ACTION {
+  EXTEND = 'extend',
+  CANCEL = 'cancel',
+  RELOAD = 'reload',
+}
+
+export const DATE_STRING_FORMAT = 'D MMM, YYYY, HH:mm:ss';
+
+export interface UISession {
```

Comment:
I'm not sure `DATE_STRING_FORMAT` and `UISession` make sense to be exported from here.

This file seems half-specific to actions, and half-general to all of Session Mgmt. How about `search/sessions_mgmt/types` for this file?